extensions/resource-deployment/src/interfaces.ts

```diff
@@ -123,20 +149,23 @@ export interface DialogInfoBase {
 	actionText?: string;
 }
 
-export interface DialogTabInfo {
+export interface DialogTabInfo extends PageInfoBase {
+}
+
+export interface PageInfoBase extends SharedFieldAttributes {
 	title: string;
 	sections: SectionInfo[];
-	labelWidth?: string;
-	inputWidth?: string;
 }
 
-export interface SectionInfo {
-	title: string;
-	fields?: FieldInfo[]; // Use this if the dialog is not wide. All fields will be displayed in one column, label will be placed on top of the input component.
-	rows?: RowInfo[]; // Use this for wide dialog or wizard. label will be placed to the left of the input component.
+export interface SharedFieldAttributes {
```

Comment:
the new names doesn't really tell what the interfaces are for, do you really need this many new interfaces? can you think of better names?

Reply:
In this case, as the name suggests this is just a collection of attributes that are shared across FeildInfo, SectionInfo, PageInfo, and WizardInfo objects. It is best that these are defined in a single place. I am certainly open to better name suggestions.